### DISCUSSION TITLE
GLTF Back Face Culling

### DISCUSSION THREAD

**Discussion by sepehr500** [visgl/deck.gl#6958] [Q&A] [answered]
How do you do set a model to blackface cull in deck.gl? Is it a parameter I pass into the scenegraph layer?

**Comment by Pessimistress** [comment, marked as answer]
https://deck.gl/docs/api-reference/core/layer#render-properties

```js
parameters: {cull: true}
```

**Reply by sepehr500** [reply]
AMAZING!
